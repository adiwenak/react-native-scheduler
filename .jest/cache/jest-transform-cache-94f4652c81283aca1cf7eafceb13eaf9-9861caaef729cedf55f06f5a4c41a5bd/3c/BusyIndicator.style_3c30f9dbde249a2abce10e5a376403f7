d8416cbc904f807a5bd8319e9912f30c
Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

var _reactNative = require("react-native");

var _default = _reactNative.StyleSheet.create({
  bottomOuterView: {
    flex: 1,
    justifyContent: "center",
    flexDirection: "row",
    alignItems: "center"
  }
});

exports.default = _default;
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi9Vc2Vycy9ha3VybmFkaS9Qcm9qZWN0cy9yZWFjdC1uYXRpdmUtc2NoZWR1bGVyL2xpYnJhcnkvQ2FsZW5kYXJNb250aC9EYXRlQ2FsZW5kYXJCb3gvSW5kaXZpZHVhbENvbXBvbmVudHMvQnVzeUluZGljYXRvci5zdHlsZS50cyJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiOzs7OztBQUFBOztlQU1lLHdCQUFXLE1BQVgsQ0FBeUI7QUFDcEMsRUFBQSxlQUFlLEVBQUU7QUFDYixJQUFBLElBQUksRUFBRSxDQURPO0FBRWIsSUFBQSxjQUFjLEVBQUUsUUFGSDtBQUdiLElBQUEsYUFBYSxFQUFFLEtBSEY7QUFJYixJQUFBLFVBQVUsRUFBRTtBQUpDO0FBRG1CLENBQXpCLEMiLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgeyBTdHlsZVNoZWV0LCBWaWV3U3R5bGUgfSBmcm9tIFwicmVhY3QtbmF0aXZlXCJcblxuaW50ZXJmYWNlIFN0eWxlIHtcbiAgICBib3R0b21PdXRlclZpZXc6IFZpZXdTdHlsZVxufVxuXG5leHBvcnQgZGVmYXVsdCBTdHlsZVNoZWV0LmNyZWF0ZTxTdHlsZT4oe1xuICAgIGJvdHRvbU91dGVyVmlldzoge1xuICAgICAgICBmbGV4OiAxLFxuICAgICAgICBqdXN0aWZ5Q29udGVudDogXCJjZW50ZXJcIixcbiAgICAgICAgZmxleERpcmVjdGlvbjogXCJyb3dcIixcbiAgICAgICAgYWxpZ25JdGVtczogXCJjZW50ZXJcIlxuICAgIH1cbn0pXG4iXX0=