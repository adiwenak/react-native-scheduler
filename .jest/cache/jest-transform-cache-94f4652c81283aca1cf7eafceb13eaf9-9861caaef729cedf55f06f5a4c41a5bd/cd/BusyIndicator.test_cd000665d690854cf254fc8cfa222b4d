09ccb81f07c6438651bbcd8eb2b3b346
var _interopRequireWildcard = require("@babel/runtime/helpers/interopRequireWildcard");

var React = _interopRequireWildcard(require("react"));

var _reactTestRenderer = require("react-test-renderer");

var _data = require("../../__stubsData/data");

var _BusyIndicator = require("./BusyIndicator");

describe("<BusyIndicator />", function () {
  it("should render with given props", function () {
    var component = (0, _reactTestRenderer.create)(React.createElement(_BusyIndicator.BusyIndicator, {
      whosBusy: _data.userCollections,
      height: 10
    })).toJSON();
    expect(component).toMatchSnapshot();
  });
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi9Vc2Vycy9ha3VybmFkaS9Qcm9qZWN0cy9yZWFjdC1uYXRpdmUtc2NoZWR1bGVyL2xpYnJhcnkvQ2FsZW5kYXJNb250aC9EYXRlQ2FsZW5kYXJCb3gvSW5kaXZpZHVhbENvbXBvbmVudHMvQnVzeUluZGljYXRvci50ZXN0LnRzeCJdLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiOztBQUFBOztBQUNBOztBQUNBOztBQUNBOztBQUVBLFFBQVEsQ0FBQyxtQkFBRCxFQUFzQixZQUFBO0FBQzFCLEVBQUEsRUFBRSxDQUFDLGdDQUFELEVBQW1DLFlBQUE7QUFDakMsUUFBTSxTQUFTLEdBQUcsK0JBQ2Qsb0JBQUMsNEJBQUQ7QUFDSSxNQUFBLFFBQVEsRUFBRSxxQkFEZDtBQUVJLE1BQUEsTUFBTSxFQUFFO0FBRlosTUFEYyxFQUlWLE1BSlUsRUFBbEI7QUFLQSxJQUFBLE1BQU0sQ0FBQyxTQUFELENBQU4sQ0FBa0IsZUFBbEI7QUFDSCxHQVBDLENBQUY7QUFRSCxDQVRPLENBQVIiLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgKiBhcyBSZWFjdCBmcm9tIFwicmVhY3RcIlxuaW1wb3J0IHsgY3JlYXRlIH0gZnJvbSBcInJlYWN0LXRlc3QtcmVuZGVyZXJcIlxuaW1wb3J0IHsgdXNlckNvbGxlY3Rpb25zIH0gZnJvbSBcIi4uLy4uL19fc3R1YnNEYXRhL2RhdGFcIlxuaW1wb3J0IHsgQnVzeUluZGljYXRvciB9IGZyb20gXCIuL0J1c3lJbmRpY2F0b3JcIlxuXG5kZXNjcmliZShcIjxCdXN5SW5kaWNhdG9yIC8+XCIsICgpID0+IHtcbiAgICBpdChcInNob3VsZCByZW5kZXIgd2l0aCBnaXZlbiBwcm9wc1wiLCAoKSA9PiB7XG4gICAgICAgIGNvbnN0IGNvbXBvbmVudCA9IGNyZWF0ZShcbiAgICAgICAgICAgIDxCdXN5SW5kaWNhdG9yXG4gICAgICAgICAgICAgICAgd2hvc0J1c3k9e3VzZXJDb2xsZWN0aW9uc31cbiAgICAgICAgICAgICAgICBoZWlnaHQ9ezEwfVxuICAgICAgICAgICAgLz4pLnRvSlNPTigpXG4gICAgICAgIGV4cGVjdChjb21wb25lbnQpLnRvTWF0Y2hTbmFwc2hvdCgpXG4gICAgfSlcbn0pXG4iXX0=