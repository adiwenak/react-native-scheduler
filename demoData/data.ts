import moment from "moment"
import { Colours } from "../library/shared/colour"
import { TaskSchedule, User, UserSelection } from "../library/shared/model"

export const user1: UserSelection = {
    id: "1",
    name: "Ben Af",
    colourIndicator: Colours.Cobalt,
    isSelected: true
}

export const user2: UserSelection = {
    id: "2",
    name: "Cardano",
    colourIndicator: Colours.Cardinal,
    isSelected: true
}

export const user3: UserSelection = {
    id: "3",
    name: "Zester",
    colourIndicator: Colours.Zest,
    isSelected: false
}

export const user4: UserSelection = {
    id: "4",
    name: "Green Lantern",
    colourIndicator: Colours.LaPalma,
    isSelected: false
}

const date1 = moment().add(1, "days")
const date2 = moment().add(2, "days")
const date3 = moment().add(3, "days")
const date4 = moment().add(4, "days")
const date5 = moment().add(1, "weeks")
const date6 = moment().add(1, "weeks").subtract(1, "days")

const date10 = moment().add(1, "month")
const date11 = moment().add(1, "month").add(1, "days")
const date12 = moment().add(1, "month").add(2, "days")

export const taskSchedules: TaskSchedule = {
    1: [
        {
            id: "1-1",
            name: "Clean Toilet",
            startTime: date1.toDate(),
            endTime: date1.add(2, "hours").toDate()
        },
        {
            id: "1-2",
            name: "Gardening",
            startTime: date2.add(1, "hours").toDate(),
            endTime: date2.add(3, "hours").toDate()
        },
        {
            id: "1-3",
            name: "Gardening",
            startTime: date6.toDate(),
            endTime: date6.add(4, "hours").toDate()
        }
    ],
    2: [
        {
            id: "2-1",
            name: "Clean Toilet",
            startTime: date1.add(3, "hours").toDate(),
            endTime: date1.add(5, "hours").toDate()
        },
        {
            id: "2-2",
            name: "Gardening",
            startTime: date2.add(2, "hours").toDate(),
            endTime: date2.add(4, "hours").toDate()
        }
    ],
    3: [
        {
            id: "3-1",
            name: "Clean Toilet",
            startTime: date3.add(1, "hours").toDate(),
            endTime: date3.add(5, "hours").toDate()
        },
        {
            id: "3-2",
            name: "Gardening",
            startTime: date4.add(2, "hours").toDate(),
            endTime: date4.add(4, "hours").toDate()
        }
    ],
    4: [
        {
            id: "4-1",
            name: "BBQ",
            startTime: date1.add(4, "hours").toDate(),
            endTime: date1.add(6, "hours").toDate()
        },
        {
            id: "4-2",
            name: "Clean Toilet",
            startTime: date5.add(1, "hours").toDate(),
            endTime: date5.add(5, "hours").toDate()
        },
        {
            id: "4-3",
            name: "Gardening",
            startTime: date6.toDate(),
            endTime: date6.add(4, "hours").toDate()
        }
    ]
}

export const allUser = [user1, user2, user3, user4]
