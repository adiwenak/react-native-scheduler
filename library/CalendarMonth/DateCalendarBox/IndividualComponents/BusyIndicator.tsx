import * as React from "react"
import {View} from "react-native"
import {Category} from "../../../shared/model"
import styles from "./BusyIndicator.style"

export interface BusyIndicatorProps {
  whosBusy: Category[]
  height: number
}

export const BusyIndicator = (props: BusyIndicatorProps): JSX.Element => {
  const circleSize = props.height / 1.7
  const circleBorderRadius = circleSize / 2
  return (
    <View
      style={[
      styles.bottomOuterView, {
        height: props.height,
        paddingBottom: 8
      }
    ]}>
      {props
        .whosBusy
        .map((obj: Category, index: number) => <View
          key={`${obj.id}-${index}`}
          style={{
          height: circleSize,
          width: circleSize,
          borderRadius: circleBorderRadius,
          marginLeft: 2,
          marginRight: 2,
          backgroundColor: obj.colour
        }}/>)
}
    </View>
  )
}
