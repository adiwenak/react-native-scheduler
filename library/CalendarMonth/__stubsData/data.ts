import moment from "moment"
import { Colours } from "../../shared/colour"
import { Activity, Category, CategoryActivities, CategorySelection } from "../../shared/model"
import { CategoryActivity } from "../CalendarMonth"

export const cat1: CategorySelection = {
  id: "1",
  name: "Ben Af",
  colour: Colours.Cobalt,
  isSelected: false
}

export const cat2: CategorySelection = {
  id: "2",
  name: "Cardano",
  colour: Colours.Cardinal,
  isSelected: false
}

export const cat3: CategorySelection = {
  id: "3",
  name: "Zester",
  colour: Colours.Zest,
  isSelected: false
}

export const cat4: CategorySelection = {
  id: "4",
  name: "Green Lantern",
  colour: Colours.LaPalma,
  isSelected: false
}

export const categoryCollections: Category[] = [cat1, cat2, cat3, cat4]

const date1 = moment().add(1, "days")
const date2 = moment().add(2, "days")
const date3 = moment().add(3, "days")
const date4 = moment().add(4, "days")
const date5 = moment().add(1, "weeks")
const date6 = moment().add(1, "weeks").subtract(1, "days")

export const allActivities: Activity[] = [
  {
    id: "1-1",
    category: "1",
    name: "Clean Toilet",
    startTime: date1.toDate(),
    endTime: date1.add(2, "hours").toDate()
  },
  {
    id: "1-2",
    category: "1",
    name: "Gardening",
    startTime: date2.add(1, "hours").toDate(),
    endTime: date2.add(3, "hours").toDate()
  },
  {
    id: "1-3",
    category: "1",
    name: "Gardening",
    startTime: date6.toDate(),
    endTime: date6.add(4, "hours").toDate()
  },
  {
    id: "2-1",
    category: "2",
    name: "Clean Toilet",
    startTime: date1.add(3, "hours").toDate(),
    endTime: date1.add(5, "hours").toDate()
  },
  {
    id: "2-2",
    category: "2",
    name: "Gardening",
    startTime: date2.add(2, "hours").toDate(),
    endTime: date2.add(4, "hours").toDate()
  },
  {
    id: "3-1",
    name: "Clean Toilet",
    category: "3",
    startTime: date3.add(1, "hours").toDate(),
    endTime: date3.add(5, "hours").toDate()
  },
  {
    id: "3-2",
    category: "3",
    name: "Gardening",
    startTime: date4.add(2, "hours").toDate(),
    endTime: date4.add(4, "hours").toDate()
  },
  {
    id: "4-1",
    category: "4",
    name: "BBQ",
    startTime: date1.add(4, "hours").toDate(),
    endTime: date1.add(6, "hours").toDate()
  },
  {
    id: "4-2",
    category: "4",
    name: "Clean Toilet",
    startTime: date5.add(1, "hours").toDate(),
    endTime: date5.add(5, "hours").toDate()
  },
  {
    id: "4-3",
    category: "4",
    name: "Gardening",
    startTime: date6.toDate(),
    endTime: date6.add(4, "hours").toDate()
  }
]

const map: CategoryActivities = {}

for (const activity of allActivities) {
  if (!map[activity.category]) {
    map[activity.category] = []
  }

  map[activity.category].push(activity)
}

const categoryActivities: CategoryActivity[] = []

Object.keys(map).forEach((categoryId: string) => {
  const category = categoryCollections.find((cat) => cat.id === categoryId)
  if (!category) {
    return
  }

  categoryActivities.push({
    category,
    tasks: map[categoryId]
  })
})

export { categoryActivities }
