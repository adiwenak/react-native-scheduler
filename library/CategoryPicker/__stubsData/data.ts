import { Category, CategorySelection, Colours } from "../../shared"

export const cat1: CategorySelection = {
    id: "1",
    name: "Ben Af",
    colour: Colours.Cobalt,
    isSelected: false
}

export const cat2: CategorySelection = {
    id: "2",
    name: "Cardano",
    colour: Colours.Cardinal,
    isSelected: false
}

export const cat3: CategorySelection = {
    id: "3",
    name: "Zester",
    colour: Colours.Zest,
    isSelected: false
}

export const cat4: CategorySelection = {
    id: "4",
    name: "Green Lantern",
    colour: Colours.LaPalma,
    isSelected: false
}

export const userSelectionCollection: CategorySelection[] = [
    cat1,
    cat2,
    cat3,
    cat4
]
