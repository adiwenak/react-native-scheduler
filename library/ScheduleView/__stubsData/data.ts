import moment from "moment"
import { Activity } from "../../shared"
import { Colours } from "../../shared/colour"

const date1 = moment().add(1, "days").set({ hour: 0, minute: 0, second: 0, millisecond: 0 })

export const dataGroup1: Activity[] = [
  {
    id: "A1-2",
    category: "A1",
    startTime: date1.set({ hour: 13, minute: 0 }).toDate(),
    endTime: date1.set({ hour: 15, minute: 0 }).toDate(),
    name: "Test A1-2"
  },
  {
    id: "A2-1",
    category: "A2",
    startTime: date1.set({ hour: 13, minute: 30 }).toDate(),
    endTime: date1.set({ hour: 15, minute: 30 }).toDate(),
    name: "Test A2-1"
  },
  {
    id: "A1-1",
    category: "A1",
    startTime: date1.set({ hour: 11, minute: 30 }).toDate(),
    endTime: date1.set({ hour: 14, minute: 30 }).toDate(),
    name: "Test A1-1"
  },
  {
    id: "A2-2",
    category: "A2",
    startTime: date1.set({ hour: 14, minute: 0 }).toDate(),
    endTime: date1.set({ hour: 16, minute: 30 }).toDate(),
    name: "Test A2-2"
  }
]

export const dataGroup2: Activity[] = [
  {
    id: "B1-2",
    category: "B1",
    startTime: date1.set({ hour: 10, minute: 0 }).toDate(),
    endTime: date1.set({ hour: 11, minute: 0 }).toDate(),
    name: "Test B1-2"
  },
  {
    id: "B2-1",
    category: "B2",
    startTime: date1.set({ hour: 9, minute: 0 }).toDate(),
    endTime: date1.set({ hour: 11, minute: 0 }).toDate(),
    name: "Test B2-1"
  },
  {
    id: "B1-1",
    category: "B1",
    startTime: date1.set({ hour: 8, minute: 30 }).toDate(),
    endTime: date1.set({ hour: 10, minute: 30 }).toDate(),
    name: "Test B1-1"
  }
]

export const dataGroup3: Activity[] = [
  {
    id: "C1-1",
    category: "C1",
    startTime: date1.set({ hour: 11, minute: 10 }).toDate(),
    endTime: date1.set({ hour: 12, minute: 30 }).toDate(),
    name: "Test C1-1"
  },
  {
    id: "C1-2",
    category: "C1",
    startTime: date1.set({ hour: 12, minute: 20 }).toDate(),
    endTime: date1.set({ hour: 13, minute: 15 }).toDate(),
    name: "Test C1-2"
  }
]

export const dataGroup4: Activity[] = [
  {
    id: "D1-1",
    category: "D1",
    startTime: date1.set({ hour: 13, minute: 40 }).toDate(),
    endTime: date1.set({ hour: 15, minute: 0 }).toDate(),
    name: "Test D1-1"
  },
  {
    id: "D2-1",
    category: "D2",
    startTime: date1.set({ hour: 11, minute: 50 }).toDate(),
    endTime: date1.set({ hour: 14, minute: 40 }).toDate(),
    name: "Test D2"
  }
]

export const dataGroup5: Activity[] = [
  {
    id: "E1-1",
    category: "E1",
    startTime: date1.set({ hour: 15, minute: 10 }).toDate(),
    endTime: date1.set({ hour: 18, minute: 20 }).toDate(),
    name: "Test E1-1"
  },
  {
    id: "E2-1",
    category: "E2",
    startTime: date1.set({ hour: 16, minute: 35 }).toDate(),
    endTime: date1.set({ hour: 17, minute: 5 }).toDate(),
    name: "Test E2-1"
  },
  {
    id: "E2-2",
    category: "E2",
    startTime: date1.set({ hour: 14, minute: 45 }).toDate(),
    endTime: date1.set({ hour: 16, minute: 25 }).toDate(),
    name: "Test E2-2"
  }
]

export const groupBackgroundColourMap = {
  A1: Colours.Indigo,
  A2: Colours.LightBlue,
  B1: Colours.Purple,
  B2: Colours.DeepPurple,
  C1: Colours.Teal,
  C2: Colours.Lime,
  D1: Colours.Yellow,
  D2: Colours.Orange,
  E1: Colours.Cardinal,
  E2: Colours.Pink
}
