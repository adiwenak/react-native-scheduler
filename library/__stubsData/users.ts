import { UserSelection, User } from '../shared';
import { Colours } from '../shared/colour'

export const user1: UserSelection = {
    id: "1",
    name: "Ben Af",
    colourIndicator: Colours.Cobalt,
    isSelected: false
}

export const user2: UserSelection = {
    id: "2",
    name: "Cardano",
    colourIndicator: Colours.Cardinal,
    isSelected: false
}

export const user3: UserSelection = {
    id: "3",
    name: "Zester",
    colourIndicator: Colours.Zest,
    isSelected: false
}

export const user4: UserSelection = {
    id: "4",
    name: "Green Lantern",
    colourIndicator: Colours.LaPalma,
    isSelected: false
}

export const userSelectionCollection: UserSelection[] = [
    user1,
    user2,
    user3,
    user4
]

export const userCollections: User[] = [
    {id: "1", name: "Adam", colourIndicator: "blue"},
    {id: "2", name: "Adi", colourIndicator: "green"},
    {id: "3", name: "Bruno", colourIndicator: "red"},
    {id: "4", name: "George", colourIndicator: "pink"},
    {id: "5", name: "Chiro", colourIndicator: "black"},
    {id: "6", name: "Olivia", colourIndicator: "grey"}
]
