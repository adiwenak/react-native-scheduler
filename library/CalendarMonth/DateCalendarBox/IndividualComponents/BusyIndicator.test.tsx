import * as React from "react"
import { create } from "react-test-renderer"
import { userCollections } from "../../__stubsData/data"
import { BusyIndicator } from "./BusyIndicator"

describe("<BusyIndicator />", () => {
    it("should render with given props", () => {
        const component = create(
            <BusyIndicator
                whosBusy={userCollections}
                height={10}
            />).toJSON()
        expect(component).toMatchSnapshot()
    })
})
