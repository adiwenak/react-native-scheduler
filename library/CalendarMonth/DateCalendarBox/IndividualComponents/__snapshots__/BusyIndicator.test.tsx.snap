// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BusyIndicator /> should render with given props 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      },
      Object {
        "height": 10,
        "paddingBottom": 8,
      },
    ]
  }
>
  <View
    style={
      Object {
        "backgroundColor": "blue",
        "borderRadius": 2.9411764705882355,
        "height": 5.882352941176471,
        "marginLeft": 2,
        "marginRight": 2,
        "width": 5.882352941176471,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "borderRadius": 2.9411764705882355,
        "height": 5.882352941176471,
        "marginLeft": 2,
        "marginRight": 2,
        "width": 5.882352941176471,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "red",
        "borderRadius": 2.9411764705882355,
        "height": 5.882352941176471,
        "marginLeft": 2,
        "marginRight": 2,
        "width": 5.882352941176471,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "pink",
        "borderRadius": 2.9411764705882355,
        "height": 5.882352941176471,
        "marginLeft": 2,
        "marginRight": 2,
        "width": 5.882352941176471,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "black",
        "borderRadius": 2.9411764705882355,
        "height": 5.882352941176471,
        "marginLeft": 2,
        "marginRight": 2,
        "width": 5.882352941176471,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "grey",
        "borderRadius": 2.9411764705882355,
        "height": 5.882352941176471,
        "marginLeft": 2,
        "marginRight": 2,
        "width": 5.882352941176471,
      }
    }
  />
</View>
`;
